README.md

```diff
@@ -4,30 +4,28 @@
 
 ### TL;DR
 
-We use this within our components to publish [statsd](http://github.com/etsy/statsd) metrics.
+We use this within our components to publish [statsd](http://github.com/etsy/statsd) metrics from .Net code.
 
-### In detail
-
-There's really not much else to say :-)
-
-We've been using this in most of our things, mostly unchanged, since around 2013.
+We've been using this in most of our things, since around 2013.
 
 ### Features
 
 * statsd metrics formatter
 * UDP client handling
 
-####
+#### Publishing statistics
+
+`IStatsDPublisher` is the interface that you will use in most circumstances. With this you can `Increment` or `Decrement` an event, and send values for a `Gauge` or `Timing`.
 
-`IStatsDPublisher` is the interface that you will use in most circumstances. Use an instance of this in order to send events, timers and gauges.
+The concrete class that implements `IStatsDPublisher` is `StatsDImmediatePublisher`. For the constructor parameters, you will need the statsd server host name. You can change the standard port (8125). You can also append a prefix to all stats. These values often come from configuration as the host name and/or prefix may vary between test and production environments.
```

Comment:
"append a prefix" sounds wrong. Prepend. We should add the ability to have both prefix and suffix, what with the difficulty we have with inconsistently named metrics...